app/views/certificates.html

```diff
@@ -122,6 +122,9 @@
 
               <!-- Server Certificate Alerts -->
               <alert ng-repeat="alert in alerts.serverKey" type="alert.type" style="margin: 20px 0px 0px 0px;">{{alert.msg}}</alert>
+              <h4><strong>Enter a passphrase (Optional)</strong></h4>
+              <input class="form-control pull-left" placeholder="Enter a passphrase here if the key requires one." type="password" ng-model="serverPassphrase" style="width: 70%;" />
```

Comment:
I would be great it we could show a symbol next to this text box to show if a password has already been submitted. See the client modal basic auth password box, Martin has done something similar there.